gm4_particles_pack/data/gm4_particles_pack/functions/load.mcfunction

```diff
@@ -1,6 +1,6 @@
-execute if score gm4 load matches 1 if score gm4_better_armour_stands load matches 1 run scoreboard players set gm4_particles_pack load 1
```

Comment:
Again, these numbers should not be altered. The website will handle the versioning